packages/pn-pa-webapp/src/utils/AppError/PAAppErrorFactory.ts

```diff
@@ -25,6 +27,12 @@ export class PAAppErrorFactory extends AppErrorFactory {
         return new GenericInvalidParameterAppError(error, this.translateFunction);
       case ServerResponseErrorCode.PN_GENERIC_INVALIDPARAMETER_DUPLICATED:
         return new GenericInvalidParameterDuplicateAppError(error, this.translateFunction);
+      // capire se devo aggiungere qui un nuovo tipo di errore per l'annullamento
+      // e creare un nuovo ts dove creo il messaggio di errore
+      case ServerResponseErrorCode.PN_NOTIFICATION_ALREADY_CANCELLED:
+        return new NotificationAlreadyCancelledAppError(error, this.translateFunction);
+      case ServerResponseErrorCode.PN_GENERIC_CANCELLED_NOTIFICATION_ERROR:
```

Comment:
Are you sure this will be the code of the error?

Reply:
I asked Simone Rondinella